frontend/src/metabase/nav/containers/MainNavbar/MainNavbarContainer/AddDataModal/tests/common.unit.spec.tsx

```diff
@@ -0,0 +1,407 @@
+import userEvent from "@testing-library/user-event";
+
+import { fireEvent, screen, within } from "__support__/ui";
+
+import { setup } from "./setup";
+
+const csvFile = new File(["test,data"], "bank-statement.csv", {
+  type: "text/csv",
+});
+const tsvFile = new File(["test,data"], "pokemon-cards.csv", {
+  type: "text/csv",
```

Comment:
this is confusing. `text/tsv` is a different file type 😕  I think maybe you just mean `otherCsvFile`